phaseone/src/test/java/HelloWorldTest.java

```diff
@@ -1,10 +1,13 @@
+import com.HelloWorldImpl;
 import org.junit.Test;
 
-public class HelloWorldTest extends HelloWorldImpl{
+public class HelloWorldTest {
```

Comment:
create another class named as HelloWorldWhileImpl and use while loop instead of for loop

test that class's method here ... additional test